File: Exemplos/App_IntentDigitalHUB_XamarinForms/Elgin_IntentDigitalHUB_XamarinForms-font/E1_Digital_Hub_Xamarin_Forms_M8/IntentServices/Bridge/ImprimirCupomNfce.cs
namespace Xamarin_Forms_Intent_Digital_Hub.IntentServices.Bridge
{
    class ImprimirCupomNfce : BridgeCommand
    {
        readonly string xml;
        readonly int indexcsc;
        readonly string csc;

        public ImprimirCupomNfce(string xml, int indexcsc, string csc) : base("ImprimirCupomNfce")
        {
            this.xml = xml;
            this.indexcsc = indexcsc;
            this.csc = csc;
        }

        protected override string FunctionParameters()
        {
            return "\"xml\"" + ":" + "\"" + xml + "\"" + "," +
                    "\"indexcsc\"" + ":" + indexcsc + "," +
                    "\"csc\"" + ":" + "\"" + csc + "\"";
        }
    }
}

File: Exemplos M10/App_IntentDigitalHUB_Ionic/Elgin_IntentDigitalHUB_Ionic-font/src/intentDigitalHubService/BRIDGE/Commands/ImprimirCupomNfce.ts
import { BridgeCommand } from "../BridgeCommand";

export class ImprimirCupomNfce extends BridgeCommand {
  readonly xml: string;
  readonly indexcsc: number;
  readonly csc: string;

  constructor(xml: string, indexcsc: number, csc: string) {
    super("ImprimirCupomNfce");
    this.xml = xml;
    this.indexcsc = indexcsc;
    this.csc = csc;
  }

  functionParameters(): object {
    return { xml: this.xml, indexcsc: this.indexcsc, csc: this.csc };
  }
}
